packages/rrweb-snapshot/src/rebuild.ts

```diff
@@ -152,7 +152,7 @@ export function buildStyleNode(
 }
 
 function buildNode(
-  n: serializedNodeWithId,
+  n: serializedNodeWithId & { chromaticAdoptedStylesheets?: string[] },
```

Comment:
I don't think we would want to keep this chromatic-specific field if submitting a PR to rrweb, but it's good enough for now

Reply:
Agreed!